requirements.txt

```diff
@@ -272,37 +281,41 @@ static3==0.5.1
     #   dj-static
 stevedore==1.29.0
     # via edx-opaque-keys
+tablib[xls,xlsx]==3.0.0
+    # via wagtail
 text-unidecode==1.2
     # via faker
 tornado==5.1.1
     # via robohash
 traitlets==4.3.2
     # via ipython
-unidecode==0.4.21
-    # via wagtail
 urllib3==1.24.2
     # via
     #   -r requirements.in
     #   botocore
     #   elasticsearch
     #   requests
     #   sentry-sdk
-uwsgi==2.0.17.1
+uwsgi==2.0.18
```

Comment:
@shaidar, @blarghmatey, Is this `uwsgi` bump alright with you guys? I tested and things are working fine for me and I just wanted to take an opinion from DevOps since it's a uwsgi change.

Additionally, We are already using `2.0.18` in [Bootcamps](https://github.com/mitodl/bootcamp-ecommerce/blob/master/requirements.txt#L326) and [MIT xPRO](https://github.com/mitodl/mitxpro/blob/master/requirements.txt#L400).

Reply:
That's fine by me. :shipit: